sdks/python/apache_beam/transforms/ptransform_test.py

```diff
@@ -476,6 +477,7 @@ def test_group_by_key(self):
       result = pcoll | 'Group' >> beam.GroupByKey() | _SortLists
       assert_that(result, equal_to([(1, [1, 2, 3]), (2, [1, 2]), (3, [1])]))
 
+  @skip('Behavior not available until Beam 2.33')
```

Comment:
Can you set the flag manually here to avoid skipping?

Reply:
Sure, did it for both tests.

At least for the first test, there was no intent to allow a flag to override the behavior, but in light of the discussion of `TriggerFn`, I've switched to having `--allow_unsafe_triggers` also override it.